src/screens/ConnectDevice/index.js

```diff
@@ -0,0 +1,280 @@
+// @flow
+import React, { Component, Fragment } from "react";
+import { SafeAreaView, StyleSheet, View } from "react-native";
+import { connect } from "react-redux";
+import { createStructuredSelector } from "reselect";
+import Icon from "react-native-vector-icons/dist/FontAwesome";
+import type { NavigationScreenProp } from "react-navigation";
+import type { Account } from "@ledgerhq/live-common/lib/types";
+
+import { accountScreenSelector } from "../../reducers/accounts";
+
+import HeaderRightClose from "../../components/HeaderRightClose";
+import StepHeader from "../../components/StepHeader";
+import Stepper from "../../components/Stepper";
+import LText from "../../components/LText";
+import CurrencyIcon from "../../components/CurrencyIcon";
+import BlueButton from "../../components/BlueButton";
+
+import colors from "../../colors";
+
+import Step from "./Step";
+
+type Props = {
+  account: Account,
+  navigation: NavigationScreenProp<{
+    params: {
+      accountId: string,
+    },
+  }>,
+};
+
+type State = {
+  deviceConnected: boolean,
+  appOpened: boolean,
+};
+
+class ConnectDevice extends Component<Props, State> {
+  static navigationOptions = ({ screenProps }: *) => ({
+    headerTitle: <StepHeader title="Device" subtitle="step 2 of 4" />,
+    headerRight: (
+      <HeaderRightClose navigation={screenProps.topLevelNavigation} />
```

Comment:
does the Close icon actually works? I think this is wrong, should be just `{ navigation }` props (historical)